### DISCUSSION TITLE
Sample CVRPTW Break Returns Error and Doesn't Work

### DISCUSSION THREAD

**Discussion by mertbakir** [google/or-tools#2447] [Routing (and legacy CP) questions] [answered]
[CVRPTW Break Sample](https://github.com/google/or-tools/blob/master/ortools/constraint_solver/samples/cvrptw_break.py) didn't work for me,

I tried it in ubuntu and windows, with Python 3.7 and OR-Tools 8.2

Windows 10: Process finished with exit code -1073741819 (0xC0000005)
Linux Ubuntu 20.04: Segmentation fault (core dumped)

**Comment by lperron** [comment, marked as answer]
duplicate of https://github.com/google/or-tools/issues/2444
